1209/B01D/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: B01D
```

Comment:
I made the change as requested to "layout: pid".  Please let me know if you
need me to submit a new pull request.

Thanks,
Justin

On Sun, Feb 7, 2016 at 12:31 PM, Nick Johnson <notifications@github.com>
wrote:

> In 1209/B01D/index.md
> <https://github.com/pidcodes/pidcodes.github.com/pull/133#discussion_r52120879>
> :
>
> > @@ -0,0 +1,8 @@
> > +---
> > +layout: B01D
>
> This needs to be "layout: pid".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pidcodes/pidcodes.github.com/pull/133/files#r52120879>
> .
>



-- 
Justin
<justin@wyolum.com>